content/en/tracing/trace_retention/_index.md

```diff
@@ -0,0 +1,93 @@
+
+For 30 days, intelligent retention retains:
+
+ - A representative selection of errors, ensuring error diversity (for example, response code 400s, 500s).
+ - High Latency in the different quartiles `p75`, `p90`, `p95`.
+ - All Resources with any traffic have associated Traces in the past for any time window selection.
+ - True maximum duration trace for each time window.
+
+**Note**: Because intelligent retention chooses spans intentionally and not randomly, spans that are retained only by the intelligent filter are **not** included in the [Trace explorer timeseries view][5]. Aggregated views (timeseries, toplist, table) are available only for spans retained by [custom retention filters](#create-your-own-retention-filter).
+
+If there are specific tags, facets, or groups of traces that you want to investigate _in detail_, meaning you want to retain more than what Intelligent Retention retains, then [create your own retention filter](#create-your-own-retention-filter). For example, you might want to keep more than a representative selection of errors from your production environment. To ensure _all_ production errors are retained and available for search and analytics for 15 days, create a 100 percent retention filter scoped to `env:prod` and `status:error`. As discussed below, this may have an impact on your bill.
+
+### Create your own retention filter
+
+{{< img src="tracing/trace_indexing_and_ingestion/IndexFilter2.mp4" style="width:100%;" alt="Span Indexing" video=true >}}
+
+To customize what spans are indexed and retained for 15 days, you can create, modify, and disable additional filters based on tags, and set a percentage of spans matching each filter to be retained. Any span that is retained has its corresponding trace saved as well, and when it is viewed, the complete trace is available. In order to be searched by tag in [Explorer][6], however, the span that directly contains the searched-upon tag must have been indexed by a retention filter.
+
+1. Name your filter.
+2. Set the tags you would like to index spans that match **all** of.
+3. Select whether this filter retains any span that matches the criteria, or only [service entry spans][4].
+4. Set a percentage of spans matching these tags to be indexed.
+5. Save your new filter.
+
+**Note:** Selecting "Top-Level Spans for Services Only" means the retention filter retains only the selected proportion of [service entry spans][4] of service and index them. Use this if you want to only index service entry spans with matching tags. If "All Spans" is selected, the retention filter retains the selected proportion of all spans of the distributed trace, irrespective of their hierarchy, and index them. This may have an impact on your bill, and the visual indicator within the app while setting a retention filter informs you how many matching spans have been detected over the time period.
```

Comment:
```suggestion
**Note**: Selecting "Top-Level Spans for Services Only" means the retention filter retains only the selected proportion of [service entry spans][4] of the service and indexes them. Use this if you want to only index service entry spans with matching tags. If "All Spans" is selected, the retention filter retains the selected proportion of all spans of the distributed trace, irrespective of their hierarchy, and indexes them. This may have an impact on your bill, and the visual indicator within the app while setting a retention filter informs you how many matching spans have been detected over the time period.
```